### PR TITLE
Add AppVeyor CI

### PR DESCRIPTION
**Pull request by Gordon-F**
Add Windows MSVC x64 build on stable, beta and nightly channel.

#14

### CODE REVIEW

**Comment by icefoxen** [comment]
@svenstaro Do we want to roll with this, or use wine under travis?  I'm kinda scared of weird platform-specific bugs, especially when they *shouldn't* happen but might anyway...

**Comment by Gordon-F** [comment]
@icefoxen wine under Travis better then windows build? 

You can't test MSVC build on Linux platform.

**Comment by svenstaro** [comment]
True about MSVC. Welp. I guess we'll go with this, then. The reason I prefer doing everything in travis is that, well, everything is on the same platform which means less management overhead.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - appveyor DownloadFile https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-devel-2.0.1-VC.zip
+    - 7z x SDL2_mixer-devel-2.0.1-VC.zip > nul
+    - xcopy SDL2_mixer-2.0.1\lib\x64\* %APPVEYOR_BUILD_FOLDER% > nul
+    - cd %APPVEYOR_BUILD_FOLDER%
+
+test_script:
+    - cargo build --verbose
+    - cargo test --verbose
+
+build: false
```
Comment (svenstaro) [inline appveyor.yml]: I'm not very knowledgeable about appveyor so: What is this line saying?

**Comment by Gordon-F** [comment]
@svenstaro Download prebuild SDL, SDL_mixer, SDL_image developer packages (prebuild only for save compile time). Unzip and copy into project folder. Compile and test.

**Comment by svenstaro** [comment]
I mean `build: false`.

**Comment by svenstaro** [comment]
Could you add that comment to the `build: false`? Afterwards I'm happy to merge.

**Comment by Gordon-F** [comment]
@svenstaro Done

**Comment by svenstaro** [comment]
Merged,  thanks a lot!

**Comment by svenstaro** [comment]
Actually this is merged now but could you perhaps add an AppVeyor badge in a separate PR?

**Comment by Gordon-F** [comment]
@svenstaro Do you enable project build at appveyor.com ?

**Comment by svenstaro** [comment]
Yes: https://ci.appveyor.com/project/svenstaro/ggez/build/job/t75kf4890s4m2dbo

**Comment by Gordon-F** [comment]
@svenstaro I can't add badge because i need security key. Please open project at appveyor.com. Go to Settings -> Badges and you can find build status badge link.

**Pull request merged**